app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    if len(command.split()) == 3:
+        action, file_name, path = command.split()
+        if action == "mv":
+            with open(file_name) as file_to_read:
+                if "/" in path:
+                    list_of_directories = path.split("/")[:-1]
+                    file_name_to_move = path.split("/")[-1]
+                    current_path = os.getcwd()
+                    for folder in list_of_directories:
+                        current_path = os.path.join(current_path, folder)
+                        if not os.path.isdir(current_path):
+                            os.mkdir(current_path)
```

Comment:
use os.makedirs

Reply:
In conditions to this task noted:
Note: You can create directory by os.mkdir("first") from module os. But to create os.mkdir("first/second"), first directory needs to exist. Function should work with different directory depth. Use os.remove() to remove the file.

if use **os.makedirs** we can ommit for loop at all. So I cant use os.path.split in this case I need all folders separetely

Reply:
Yes, but `use os.makedirs` can help you with in this case